### DISCUSSION TITLE
Skupper service IP changes without manual intervention - what is the criteria for this to happen

### DISCUSSION THREAD

**Discussion by nmcconom** [skupperproject/skupper#1506] [Q&A]
We have seen on occasion that the Skupper service IP has changed in our cluster a few times after creation.

It looks like this has happened when IP connectivity between the two clusters/links has had problems for whatever reason.

For most of our components that is fine as they are using the service name and resolve to the new IP address - there are a couple of legacy items/scripts that are still relying on IP though for now and we just wondered what exactly is the criteria for the services to be re-created and get new IPs? - couldn't find it documented anywhere

Thanks in advance.

**Comment by grs** [comment]
It happens when sites are dosconnected for some time and the services that were exposed on one site are then removed from the other site(s). Then when connectivity is re-established the exposing site readvertises the services and they get recreated in  the other site(s).
